pkg/helm/environment/environment.go

```diff
@@ -59,6 +61,7 @@ func (s *EnvSettings) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.Debug, "debug", false, "enable verbose output")
 	fs.StringVar(&s.TillerNamespace, "tiller-namespace", "kube-system", "namespace of Tiller")
 	fs.Int64Var(&s.TillerConnectionTimeout, "tiller-connection-timeout", int64(300), "the duration (in seconds) Helm will wait to establish a connection to tiller")
+	fs.BoolVar(&s.SkipSelfUpdateCheck, "skip-self-update-check", false, "disable checking for updates to Helm")
```

Comment:
double negatives always throw me off. What do you think about re-naming this to `check-for-updates` or something, and changing the default to true?

Reply:
How would someone set it to false at the command line? How would the internal processing map the CLI value to the internal variable?

Do you have a good example of making it a positive?

Reply:
Yeah! The variable name could change to `s.PerformSelfUpdateCheck`, which is set to `true` by default.

Then, on line 98 in helm.go we'd change `if !settings.SkipSelfUpdateCheck` to `if settings.PerformSelfUpdateCheck`. The `!settings.SkipSelfUpdateCheck` to `if settings.PerformSelfUpdateCheck` is the double negative I'm referencing if it wasn't totally clear btw. Does that help make sense or do you wanna chat about this offline?

> How would someone set it to false at the command line?

`helm --check-for-updates=false`